deployer/deployer.yaml

```diff
@@ -323,4 +329,16 @@ items:
     description: "(Deprecated) Node selector operations Curator (label=value)."
     name: CURATOR_OPS_NODESELECTOR
     value: ""
+  -
+    description: 'Use the systemd journal for the fluentd log source'
+    name: USE_JOURNAL
+    value: "false"
```

Comment:
The default should be - whatever docker is set up to use.  If we are deploying logging (fluentd) on a system which has docker `--log-driver=journald` we should configure fluentd to read from journald.  So I guess I need to add logic to the deployer and logging.sh to do that.